index.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+const imagemin = require('imagemin');
+const imageminMozjpeg = require('imagemin-mozjpeg');
+const imageminPngquant = require('imagemin-pngquant');
+const Svgo = require('svgo');
+const fs = require('fs');
+const os = require('os');
+const async = require('async');
+const path = require('path');
+
+const compress = (file, level, callback) => {
```

Comment:
If we're planning on expanding the things that optimiz can do, each operation should be in its own file.